source/developers/extending-studio/external-control.rst

```diff
@@ -0,0 +1,187 @@
+:is-up-to-date: True
```

Comment:
Please rename to "Building Form Engine Plugins"

Please refer to the capability as plugins instead of external.